gradle/verification-metadata.xml

```diff
@@ -895,6 +895,11 @@
             <pgp value="0191e61acbbe76323ac15c83b5ad94bdd6bdb924"/>
          </artifact>
       </component>
+      <component group="me.champeau.jmh" name="jmh-gradle-plugin" version="0.6.8">
```

Comment:
Can the entry above be removed then?

Reply:
I'll check

Reply:
I removed the previous jmh versions from this file.